templates/1_set_up_toffy.ipynb

```diff
@@ -235,7 +235,8 @@
     "# create a new folder with the sweep name\n",
```

Comment:
<p>This is good, but the information about how they are supposed to select the cutoff is buried in the middle (where the counts level off). I would maybe put this as a separate paragraph, or have it come first. Something so that it's more obvious</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/250/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>